docs/source/reservations/reservations.rst

```diff
@@ -65,31 +74,34 @@ For example, if you are launching a cluster with the following SkyPilot YAML:
       cloud: aws
       accelerators: A100:8
     
-    num_nodes: 4
+    num_nodes: 2
 
 
 SkyPilot will utilize the capacity reservation as follows:
-1. Find available capacity in your capacity block in ``us-west-2`` in the reservation ``cr-123456789a``. Let's say 1 A100 instance capacity is available.
-2. SkyPilot will launch 4 nodes in ``us-west-2`` with 1 node coming from the reservation, and the rest 3 nodes from on-demand.
-3. If it fails to find available A100 instances from on-demand, it will automatically :ref:`failover <auto-failover>` to other regions/zones.
+
+1. Find available capacity in your capacity blocks in ``us-east-2`` and ``us-west-2`` in the reservation ``cr-123456789a`` and ``cr-23456789ab``, respectively. Let's say 1 A100 instance capacity is available in ``us-east-2`` but no available capacity in ``us-west-2`` (as shown in the figure above).
```

Comment:
Not sure I fully understand this list based semantics (mentioned before). E.g., it seems step 3 needs to be expanded to say, failover to other places to look at cr-234....

Also is it guaranteed to look at cr-234... next?

Reply:
Updated the steps. It is not guaranteed to go to `cr-234` as it does not have any available capacity.